### PR TITLE
St feature lesson screen

### PR DESCRIPTION
**Pull request by saveentoor**
Edit and finished styling for one component in Lessons (My lessons)

### CODE REVIEW

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Is this a duplicate pull request? Please resolve merge conflicts on this branch. Thank you.

**Comment by teelau** [comment]
Rebase this branch after the changes for the SurveyDifficulty are merged, this branch contains changes from that feature and the new lessons screen feature.

**Comment by teelau** [comment]
Connect with an issue

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts for this branch and provide valid labels and milestone.

File: client/js/screens/Lessons/LessonsContainer.js [hunk 1]
```diff
@@ -1,14 +1,128 @@
...
+              >
+                Library
+              </Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+
+        <View>
+          {this.state.tabIndex == 0 ? <Text>test 1</Text> : <Text>test2</Text>}
+        </View>
```
Comment (teelau) [inline client/js/screens/Lessons/LessonsContainer.js]: Remove this testing stuff

**Review by teelau — CHANGES_REQUESTED** [review]

File: client/js/screens/Lessons/LessonsContainer.js [hunk 2]
```diff
@@ -1,14 +1,128 @@
...
+        <View style={styles.editButton}>
+          <TouchableOpacity style={styles.advancedButton}>
+            <Text style={styles.buttonColor}>Edit</Text>
+          </TouchableOpacity>
+        </View>
+
+        <View style={styles.container}>
+          <View style={[{ backgroundColor: "#99c355" }, styles.iconContainer]}>
```
Comment (siddharthparmar7) [inline client/js/screens/Lessons/LessonsContainer.js]: move this background color to the styles.js file

File: client/js/screens/Lessons/LessonsContainer.js [hunk 3]
```diff
@@ -1,14 +1,128 @@
...
+          </View>
+        </View>
+        <View style={styles.addButtonContainer}>
+          <TouchableOpacity>
+            <Image
+              style={styles.addButton}
+              source={require(`../../assets/images/Buttons/add.png`)}
+            />
+          </TouchableOpacity>
+        </View>
```
Comment (siddharthparmar7) [inline client/js/screens/Lessons/LessonsContainer.js:120]: Please move all your styles to styles.js.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes. Thank you.

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**